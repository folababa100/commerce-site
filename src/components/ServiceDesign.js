import React from 'react';

const ServiceDesign = () => {
  return (
    <div className="container">
      <div className="row">
        <div className="col-12 col-sm-8 col-md-6">
          <div className="text-center">
            <h3>Welcome to our service page</h3>
          </div>
        </div>
      </div>
    </div>
  )
}