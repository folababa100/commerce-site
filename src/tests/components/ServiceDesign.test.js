import React from 'react';
import { shallow } from "enzyme";
import ServiceDesign from '../../components/ServiceDesign';

test('should correctly render component', () => {
  const wrapper = shallow(<ServiceDesign/>);
  expect(wrapper).toMatchSnapshot();
})

