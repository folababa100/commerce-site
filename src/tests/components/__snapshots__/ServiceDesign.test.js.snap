// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render component 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-12 col-col-md-8 col-lg-6"
    >
      <div
        className="text-center"
      >
        <h3>
          Welcome to our service page
        </h3>
      </div>
    </div>
  </div>
</div>
`;
